api/src/main/kotlin/nl/avisi/anowalke/controllers/RestController.kt

```diff
@@ -0,0 +1,39 @@
+package nl.avisi.anowalke.controllers
+
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import mu.KotlinLogging
+import nl.avisi.anowalke.dto.ExpressionsDto
+import nl.avisi.anowalke.services.FilterService
+import nl.avisi.anowalke.services.VertaalService
+
+@RestController
+@RequestMapping("/")
+class RestController(
+    private val vertaalService: VertaalService,
+    private val filterService: FilterService
+) {
+
+    private val log = KotlinLogging.logger { }
+
+    @PostMapping(value = ["/translate"])
+    fun translate(
+        @RequestBody expressions: ExpressionsDto
+    ): List<String> {
```

Comment:
Voor het ontvangen van de expressions heb je een data transfer object aangemaakt. Dat bevat een lijst met strings. Nu geef je die hier wel zomaar terug. Wil je daar ook nog een DTO voor introduceren?

Reply:
Hmmm dat vind ik een lastige is het raar als ik daar ook een expressionsDto terug zou geven of moet ik daar dan een extra precies hetzelfde object voor maken?

Reply:
Denk dat het niet gek ik als je dezelfde soort data wilt terugsturen.